client/components/tinymce/plugins/editor-button-analytics/plugin.js

```diff
@@ -79,8 +74,8 @@ function editorButtonAnalytics( editor ) {
 			// This is a menu item in the format dropdown.  Only track which
 			// specific item is clicked for english interfaces - the easiest
 			// way to determine which item is selected is by UI text.
-			const currentUser = user.get();
-			const locale = currentUser ? currentUser.localeSlug : 'en';
+			const reduxStore = editor.getParam( 'redux_store' );
+			const locale = reduxStore ? getCurrentUserLocale( reduxStore.getState() ) : null;
```

Comment:
`en` would be a better default than `null` here, as it was in the original code. With `null`, analytics events will be reported as `format_dropdown_non_english`. Not that it would matter much, I think it's pretty much guaranteed that the `redux_store` is always there.